engine/access/apiproxy/access_api_proxy.go

```diff
@@ -28,9 +29,7 @@ type FlowAccessAPIRouter struct {
 	Logger   zerolog.Logger
 	Metrics  *metrics.ObserverCollector
 	Upstream *FlowAccessAPIForwarder
-
-	// local observer
-	access.AccessAPIServer
+	Observer *protocol.Handler
```

Comment:
I mean here, why did we change it to Observer?

Reply:
If this file is only used by observers, then can we move the file somewhere else, because the current location is confusing.

On the other hand if this file is used by both access node and observer, then we should rename this field to something more generic

Reply:
The access_api_proxy.go file name could be changed. I don't think it should be part of the scope for this PR though. I changed `access.AccessAPIServer` to Observer since the Observer should not support all RPCs.

Reply:
But is this file used by the observer only? or is it also used by Access node?

Reply:
Only the observer uses it